### PR TITLE
Send total counts of AHM of Project, Package and Requests

### PR DESCRIPTION
**Pull request by mdeniz**
For showing totals for projects, packages and requests while are
created/deleted we needed to send the counts

### CODE REVIEW

File: src/api/app/models/event/create_package.rb [hunk 1]
```diff
@@ -19,7 +19,7 @@ def metric_tags
     end
 
     def metric_fields
-      { value: 1 }
+      { project: payload['project'], package: payload['package'], count: Package.count }
```
Comment (Ana06) [inline src/api/app/models/event/create_package.rb]: why `payload['project']`? didn't we say we don't want this info there? :thinking:

Comment (mdeniz) [reply src/api/app/models/event/create_package.rb]: Yes, but we change our mind :grinning:

Comment (Ana06) [reply src/api/app/models/event/create_package.rb]: we? ... you! explain me why! :wink:

Comment (mdeniz) [reply src/api/app/models/event/create_package.rb]: We will be able to extract the info from influxdb to another tool as you said when we started to discuss this :grin: :laughing:

Comment (Ana06) [reply src/api/app/models/event/create_package.rb]: Ah! If I was right... then I'll approve it.... :laughing:

File: src/api/app/models/event/create_project.rb [hunk 2]
```diff
@@ -19,7 +19,7 @@ def metric_tags
     end
 
     def metric_fields
-      { value: 1 }
+      { project: payload['project'], count: Project.count }
```
Comment (hennevogel) [inline src/api/app/models/event/create_project.rb]: Why do you replace it? Why don't you send both?

Comment (ChrisBr) [reply src/api/app/models/event/create_project.rb]: I would put the project as tags. Tags are supposed to be finite but it doesn't matter if you have a large amount of tags. Tags are indexed and very performant. Then you can do queries like ```where project = Factory```

**Comment by ChrisBr** [comment]
So while thinking about the tags:

- Create package: project and user (if available) as tags
- Create project: user (if available) as tags
- Request state change: target package and target project as tags
- Delete package: project and user (if available) as tags
- Delete project: user (if available) as tags

**Comment by ChrisBr** [comment]
Talked in person: scratch the user again!

**Comment by mdeniz** [comment]
Grafana can only count what you send... so, creation and deletion of projects/packages or state changes of requests. But requests that are not changing or projects/packages that are already there... you can't know how many of them are.

**Comment by bgeuken** [comment]
Right, but counting all requests every time a request gets created sounds like an overkill. Do we even need a counter of all requests (or packages or projects) ever created?

**Comment by Ana06** [comment]
@mdeniz what is the state of this? :confused:

**Comment by hennevogel** [comment]
@mdeniz?

**Comment by mdeniz** [comment]
@hennevogel ?
What do you think about what @bgeuken says??

**Comment by hennevogel** [comment]
>  What do you think about what @bgeuken says??

I think he is right :-) But that would need https://github.com/GoshPosh/grafana-meta-queries I think

**Comment by hennevogel** [comment]
We talked about this offline and want to try the plugin instead...

**Comment by mdeniz** [comment]
Ok, let's close this PR, we will try other approach to this.

**Pull request closed without merging**